phpunit/class-gutenberg-rest-templates-controller-test.php

```diff
@@ -131,23 +131,25 @@ public function test_get_item() {
 
 		$this->assertSame(
 			array(
-				'id'             => 'emptytheme//my_template',
-				'theme'          => 'emptytheme',
-				'slug'           => 'my_template',
-				'source'         => 'custom',
-				'origin'         => null,
-				'type'           => 'wp_template',
-				'description'    => 'Description of my template.',
-				'title'          => array(
+				'id'              => 'emptytheme//my_template',
+				'theme'           => 'emptytheme',
+				'slug'            => 'my_template',
+				'source'          => 'custom',
+				'origin'          => null,
+				'type'            => 'wp_template',
+				'description'     => 'Description of my template.',
+				'title'           => array(
```

Comment:
Confirmed that these changes are not backported to [Core version of this test file](https://github.com/WordPress/wordpress-develop/blob/5bd13e7cc1a2f7b649c1b97e3ada88b8950ed2b6/tests/phpunit/tests/rest-api/wpRestTemplatesController.php#L148-L170).